executor_manager/services/task_heartbeat_manager.py

```
@@ -0,0 +1,573 @@
+                error_message=error_message,
+                executor_name=executor_name,
+            )
+            if success:
+                logger.info(
+                    f"[RunningTaskTracker] Marked task {task_id} as failed via Backend API"
+                )
+            else:
+                logger.warning(
+                    f"[RunningTaskTracker] Failed to mark task {task_id} as failed: {result}"
+                )
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)